src/sinks/tsd-query-log-sink.ts

```diff
@@ -72,29 +74,35 @@ export class TsdQueryLogSink implements tsdDef.Sink {
     private static stringify(metricsEvent:tsdDef.MetricsEvent):string {
         var transformedMetricsEvent = TsdQueryLogSink.transformMetricsEvent(metricsEvent);
 
-        return JSON.stringify(utils.stenofy(transformedMetricsEvent),
+        return JSON.stringify(utils.stenofy(transformedMetricsEvent, transformedMetricsEvent.annotations["_host"]),
```

Comment:
Unclear why the host is extracted outside the function call, this seems like logic you would want in that method. No?